india_compliance/gst_india/doctype/gst_settings/gst_settings.json

```diff
@@ -1,264 +1,265 @@
-   "fieldname": "auto_generate_e_waybill",
-   "fieldtype": "Check",
-   "label": "Automatically Generate e-Waybill on Invoice Submission",
-   "read_only_depends_on": "eval: doc.enable_e_invoice && doc.auto_generate_e_invoice"
-  },
-  {
-   "default": "1",
-   "depends_on": "eval: doc.enable_api && doc.enable_e_waybill",
-   "description": "Automatically fetch e-Waybill data after it's generation for printing or logging purposes. This will lead to one additional API request.",
-   "fieldname": "fetch_e_waybill_data",
-   "fieldtype": "Check",
-   "label": "Fetch e-Waybill Data After Generation",
-   "read_only_depends_on": "eval: doc.attach_e_waybill_print"
-  },
-  {
-   "default": "1",
-   "depends_on": "eval: doc.enable_e_invoice",
-   "description": "e-Invoice will be automatically generated after Sales Invoice submission if applicable",
-   "fieldname": "auto_generate_e_invoice",
-   "fieldtype": "Check",
-   "label": "Automatically Generate e-Invoice on Invoice Submission"
-  },
-  {
-   "default": "0",
-   "fieldname": "enable_e_invoice",
-   "fieldtype": "Check",
-   "label": "Enable e-Invoicing"
-  },
-  {
-   "default": "0",
-   "fieldname": "enable_reverse_charge_in_sales",
-   "fieldtype": "Check",
-   "label": "Enable Reverse Charge in Sales Transactions"
-  },
-  {
-   "default": "0",
-   "depends_on": "eval: doc.enable_e_waybill",
-   "fieldname": "enable_e_waybill_from_dn",
-   "fieldtype": "Check",
-   "label": "Enable e-Waybill Generation from Delivery Note"
-  },
-  {
-   "default": "0",
-   "description": "Fields and validations for these transactions will apply based on this setting",
-   "fieldname": "enable_overseas_transactions",
-   "fieldtype": "Check",
-   "label": "Enable SEZ / Overseas Transactions"
-  },
-  {
-   "depends_on": "eval: doc.enable_api",
-   "fieldname": "other_apis_section",
-   "fieldtype": "Section Break",
-   "label": "Other APIs"
-  },
-  {
-   "default": "1",
-   "description": "When creating a new Customer / Supplier / Address using the Quick Entry form, other fields will be automatically fetched based on the GSTIN entered",
-   "fieldname": "autofill_party_info",
-   "fieldtype": "Check",
-   "label": "Autofill Party Information based on GSTIN"
-  }
- ],
- "index_web_pages_for_search": 1,
- "issingle": 1,
- "links": [],
- "modified": "2022-06-29 07:46:51.237057",
- "modified_by": "Administrator",
- "module": "GST India",
- "name": "GST Settings",
- "owner": "Administrator",
- "permissions": [
-  {
-   "read": 1,
-   "role": "All"
-  },
-  {
-   "create": 1,
-   "read": 1,
-   "role": "Accounts Manager",
-   "write": 1
-  },
-  {
-   "create": 1,
-   "read": 1,
-   "role": "Accounts User",
-   "write": 1
-  },
-  {
-   "create": 1,
-   "read": 1,
-   "role": "System Manager",
-   "write": 1
-  }
- ],
- "quick_entry": 1,
- "sort_field": "modified",
- "sort_order": "DESC",
- "states": [],
- "track_changes": 1
+    "actions": [],
+    "creation": "2022-06-29 07:45:13.261573",
```

Comment:
Make sure formatting changes are not a part of your PR. Revert formatting changes

Reply:
Done